package.json

```diff
@@ -17,7 +17,9 @@
     "format-check": "prettier --check .",
     "format-check:rust": "turbo run format-check:rust",
     "playwright-install": "playwright install",
-    "all-check": "pnpm install && pnpm compile && pnpm format-check && pnpm lint && pnpm test && pnpm download-keys && pnpm build && pnpm format-check:rust && pnpm lint:rust && pnpm test:rust"
+    "changeset": "changeset",
+    "changeset:publish": "changeset publish",
+    "all-check": "pnpm install && pnpm clean && pnpm compile && pnpm format-check && pnpm lint && pnpm test && pnpm download-keys && pnpm build && pnpm format-check:rust && pnpm lint:rust"
```

Comment:
i think having clean in all-check makes dev difficult because it blows away keys and results in a lot of redownload